File: app/src/main/java/com/qwe7002/telegram_rc/data_structure/polling_json.java
package com.qwe7002.telegram_rc.data_structure;

public class polling_json {
    @SuppressWarnings({"unused", "RedundantSuppression"})
    public final String[] allowed_updates = {"message", "channel_post", "callback_query"};
    public long offset;
    public int timeout;
}

File: app/src/main/java/com/qwe7002/telegram_rc/data_structure/PollingJson.kt
package com.qwe7002.telegram_rc.data_structure;
@SuppressWarnings({"unused", "RedundantSuppression"})
public class pollingJson {
    public final String[] allowed_updates = {"message", "channel_post", "callback_query"};
    public long offset;
    public int timeout;
}
